src/qt/forms/debugwindow.ui

```diff
@@ -657,6 +657,181 @@
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
+           <item>
+            <widget class="QGroupBox" name="groupBox_2">
+             <property name="toolTip">
+              <string extracomment="Totals"/>
```

Comment:
How does a user see this tooltip?

Reply:
thanks - removed

Reply:
The traffic history always took time to repaint.
I doubt there is an easy fix - it would require caching excessive data in anticipation of a resolution change.
I would lean toward prudent disk space usage over this inconvenience.

Reply:
Thanks, makes sense and is unrelated to this change. Will re-review tomorrow.